testsuite/tests/lib-reachable/testopt.ml

```diff
@@ -0,0 +1,69 @@
+(* TEST
+   * native
+*)
+
+(* todo: TEXSTXX
+
+include dynlink
+libraries = ""
+files = "plugin.ml testopt.ml"
+
+* native
+** shared-libraries
+*** setup-ocamlopt.opt-build-env
+**** ocamlc.opt
+program = "${test_build_directory}/testopt.opt"
+flags = "-linkall"
+include ocamlcommon
+libraries += "dynlink"
+all_modules = "testopt.ml"
+***** run
+******check-program-output
+*)
+
```

Comment:
I wanted to load the code in `plugin.ml` dynamically but couldn't figure it out--not having used the test suite or dynamic loading before.  The variable defined in `plugin.ml` would test the code path for dynamic roots.